Cargo.toml

```diff
@@ -70,6 +70,8 @@ defmt = ["dep:defmt", "heapless/defmt-03"]
 "socket-dns" = ["socket", "proto-dns"]
 "socket-mdns" = ["socket-dns"]
 
+"socket-tcp-cubic" = [] # Enable Cubic TCP congestion control algorithm
```

Comment:
I'd like all congestion control algorithms to be pluggable since this is what we also use for all other features.

Reply:
`socket-tcp-reno` feature has been added.